src/app/server/Server.h

```diff
@@ -678,6 +689,8 @@ class Server
 
     Access::AccessControl mAccessControl;
     app::AclStorage * mAclStorage;
+    Access::AccessRestriction * mAccessRestriction;
```

Comment:
wrap this in #if CHIP_CONFIG_USE_ACCESS_RESTRICTIONS?

Reply:
done